client/my-sites/stats/stats-comments/style.scss

```diff
@@ -13,3 +18,53 @@
 .tab-top-content .tab-content.top-content {
 	display: block;
 }
+
+.stats__modernised-comments {
+	.segmented-control {
+		.segmented-control__link {
+			padding: 6px 8px;
+			font-weight: 500;
+			line-height: 20px;
+			color: var(--studio-black);
+
+			&:focus {
+				outline: none;
+			}
+		}
+
+		.segmented-control__item {
+			&:not(:first-of-type) {
+				margin-left: -1px;
+			}
+
+			&.is-selected {
+				.segmented-control__link {
```

Comment:
We need to account for the "Following" text to ensure sufficient contrast for legibility:

<img width="627" alt="image" src="https://user-images.githubusercontent.com/4044428/218193379-7141ba38-ba41-4d92-854a-4577fe7e8d74.png">